pkg/cloud/services/networking/securitygroups.go

```diff
@@ -25,12 +25,15 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/pkg/record"
 )
 
+// Exported Constants can be used to calculate securitygroupnames in other packages
 const (
-	secGroupPrefix     string = "k8s"
-	controlPlaneSuffix string = "controlplane"
-	workerSuffix       string = "worker"
-	bastionSuffix      string = "bastion"
-	remoteGroupIDSelf  string = "self"
+	secGroupPrefix                string = "k8s"
+	ControlPlaneSuffix            string = "controlplane"
+	WorkerSuffix                  string = "worker"
+	BastionSuffix                 string = "bastion"
+	NeutronLbaasSuffix            string = "lbaas"
```

Comment:
ControlPlaneSuffix, WorkerSuffix, BastionSuffix, NeutronLbaasSuffix should start with small character: controlPlaneSuffix, workerSuffix, bastionSuffix, neutronLbaasSuffix respectively.

Reply:
NeutronLbaasSuffix is exported to loadbalancer package. ControlPlaneSuffix, WorkerSuffix, BastionSuffix are not exported in my understanding. Let me know if I am wrong.